gotime/go-time-30.md

```diff
@@ -402,7 +402,7 @@ I've shown Erik a video and Erik's like, "I don't see anything", and I'm like, "
 
 **Erik St. Martin:** Awesome! I need to remember that, because I've always been looking for apps to do window management when I'm on my Mac. Alright, who's next?
 
-**Carlisia Pinto:** I can go next. This week somebody showed me a tool that is like Go Playground, but for Docker, and it's really cool. This tool was made my Marcos \[unintelligible 01:13:16.15\] He's from Argentina. He did this tool - it's basically a virtual machine on the web, where you can run Docker containers and create clusters with Docker features, like \[unintelligible 01:13:33.29\]. You can also pair-program... It's really cool, you should check it out. Did you guys know this tool? I'm gonna paste the link. I had never seen it before.
+**Carlisia Pinto:** I can go next. This week somebody showed me a tool that is like Go Playground, but for Docker, and it's really cool. This tool was made my [Marcos Nils](https://www.twitter.com/marcosnils). He's from Argentina. He did this tool - it's basically a virtual machine on the web, where you can run Docker containers and create clusters with Docker features, like swarm modes. You can also pair-program... It's really cool, you should check it out. Did you guys know this tool? I'm gonna paste the link. I had never seen it before.
```

Comment:
A small piece after Marcos' name was dropped, can you add it? I also fixed a small typo here:
> This tool was made by Marcos Nils... I'm sure I'm not saying his last name correctly.

Reply:
Sure! 

The reason I added the Marcos' Twitter profile is that the Readme under Text formatting mentioned adding links to people and projects. I did go through a couple of the other podcasts, specifically Go Time 36-38, where the profile links were added.